Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM python:3.7
+FROM python:3.9
 COPY ./requirements.txt /dockerBuild/requirements.txt
 WORKDIR /dockerBuild
 RUN pip install --no-cache-dir -r requirements.txt
 COPY . /dockerBuild
-CMD ["flask", "run", "--host", "0.0.0.0"]
+CMD ["poetry", "run", "start"]
```

Comment:
Let this be. Docker is containerized anyways.

Reply:
I guess, that `poetry start` runs only `python run.py`, and that isn't what was originally in the Dockerfile, so it will probably change the behavior of how this works 🤔